assests/javaScript/home.js

```diff
@@ -0,0 +1,39 @@
+const movieList = document.getElementById("movie-list");
+let markUp = "";
+
+fetch(
+  `https://api.themoviedb.org/3/movie/top_rated?api_key=d2aa1d6a7ef9316cd7e2270dd937b843&language=en-US&page=1`
```

Comment:
It's better to define api key as separate variable.

Reply:
Done